File: memories/2024/exercise/java/day13/src/main/java/santamarket/model/Receipt.java
package santamarket.model;

public class Receipt {
    private double totalPrice = 0.0;
    private double totalDiscount = 0.0;

    public void addProduct(double price) {
        totalPrice += price;
    }

    public void addDiscount(double discount) {
        totalDiscount += discount;
    }

    public double getTotalPrice() {
        return totalPrice - totalDiscount;
    }
}

File: memories/2024/exercise/kotlin/day13/src/main/kotlin/santamarket/model/Receipt.kt
package santamarket.model

class Receipt {
    private var totalPrice = 0.0
    private var totalDiscount = 0.0

    fun addProduct(price: Double) {
        totalPrice += price
    }

    fun addDiscount(discount: Double) {
        totalDiscount += discount
    }

    fun getTotalPrice(): Double = totalPrice - totalDiscount
}
